FirstPersonScienceApp.vcxproj

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Maybe we want to consider keeping `FirstPersonScience` as the application (rather than renamed to `FirstPersonScienceApp` and rename what is `FirstPersonScience` above to something like `FPSciLib` or something like that?

Reply:
Sounds good to me. Should we keep everything `FirstPersonScience` or `FPSci` as well?